### PR TITLE
add networkwatcher extension schema

### PR DESCRIPTION
**Pull request by jluk**
@TianoMS Could we get this merged in for the networkwatcher release?

### CODE REVIEW

**Comment by gurudennis** [comment]
Looks good to me. Thanks.

**Comment by TianoMS** [comment]
Change looks good. Can you add one test for it?

**Comment by jluk** [comment]
@TianoMS Done!

File: tests/2015-08-01/Microsoft.Compute.tests.json [hunk 1]
```diff
@@ -1022,6 +1022,40 @@
...
+      "name": "extensions - Test networkWatcherAgentWindows",
+      "definition": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Compute.json#/resourceDefinitions/extensions",
+      "json": {
+        "name": "Test Name",
+        "location": "Test Location",
+        "type": "Microsoft.Compute/virtualMachines/extensions",
+        "apiVersion": "2015-06-15",
+        "properties": {
+          "publisher": "Microsoft.Azure.NetworkWatcher",
+          "type": "networkWatcherAgentWindows",
```
Comment (TianoMS) [inline tests/2015-08-01/Microsoft.Compute.tests.json]: It's case sensitive and should be "NetworkWatcherAgentWindows". Same below.

**Comment by jluk** [comment]
@TianoMS Fixed and reran, thanks!

**Comment by gurudennis** [comment]
Thank you!

**Pull request merged**